lib/fluent/output.rb

```diff
@@ -450,9 +450,13 @@ def initialize
       super
     end
 
+    def format_stream(tag, es)
+      es.to_msgpack_stream
+    end
+
     def emit(tag, es, chain)
       @emit_count += 1
-      data = es.to_msgpack_stream
+      data = format_stream(tag, es)
```

Comment:
Extracting routine to `format_stream` is for `@time_as_integer` so this diff itself is not needed for this patch?
We want to reduce the change without no reason.